src/test/article.spec.js

```diff
@@ -4,10 +4,12 @@ import sinon from 'sinon';
 import app from '../index';
 import BaseRepository from '../repository/base.repository';
 import { createUser, createArticle, generateArticle } from './utils/helpers';
+import { articleSample, articleSample2 } from './mockdata/articledata';
```

Comment:
Please use the `createArticle`/`generateArticle` method here to generate your mock articles

Reply:
Thank you for this feedback.
The methods you suggested is not effective to test some of the required behavior of the application using a TDD approach